src/promptflow-evals/tests/evals/e2etests/test_metrics_upload.py

```diff
@@ -102,6 +102,9 @@ def test_logging_metrics(self, setup_data, caplog):
     @pytest.mark.usefixtures("vcr_recording")

```

Comment:
Ideally we don't want to touch the recording file after the recording. if vcr recording framework doesn't serve our need and require maunal step, let's just disable the vcr for that test for now.

Reply:
Fixed